src/gui/toolbarMenubar/ColorToolItem.h

```diff
@@ -23,7 +23,7 @@ class ColorSelectImage;
 
 class ColorToolItem: public AbstractToolItem {
 public:
-    ColorToolItem(ActionHandler* handler, ToolHandler* toolHandler, GtkWindow* parent, Color color,
+    ColorToolItem(ActionHandler* handler, ToolHandler* toolHandler, GtkWindow* parent, const NamedColor& namedColor,
```

Comment:
You could move the value in the constructor.

```suggestion
    ColorToolItem(ActionHandler* handler, ToolHandler* toolHandler, GtkWindow* parent, NamedColor namedColor,
```

Reply:
I think I'm probably missing something, namely the same thing as in https://github.com/xournalpp/xournalpp/pull/2379#discussion_r548090298.

Reply:
Question resolved. Will be done :heavy_check_mark: